proving_system/stark/benches/functions/stark.rs

```diff
@@ -15,14 +20,20 @@ use lambdaworks_stark::air::example::{
 
 use crate::util::FE;
 
-pub fn prove_fib(trace_length: usize) {
+#[allow(dead_code)]
```

Comment:
Shouldn't need a dead code here. See if you need to compile for test or reduce visibility